backend/urls.py

```diff
@@ -27,6 +27,7 @@
     path('poll/user/schedule', PollUserScheduleListAPI.as_view(), name='poll_user_schedule'),
     path('schema/', SpectacularAPIView.as_view(), name='schema'),
     path('schema/redoc/', SpectacularRedocView.as_view(url_name='api:schema'), name='redoc'),
+    path('comments/', include((comment_patterns, 'comment'))),
```

Comment:
Comment patterns already exist for respective module that uses it e.g. group/urls.py, poll/urls.py

Reply:
I have removed the changes